Classes/FieldProcessor/Service.php

```diff
@@ -92,6 +92,18 @@ public function processDocument(Document $document, array $processingConfigurati
                     case 'uppercase':
                         $fieldValue = array_map('mb_strtoupper', $fieldValue);
                         break;
+                    default:
```

Comment:
I think code should be
`if (in_array('ApacheSolrForTypo3\Solr\FieldProcessor\FieldProcessor', class_implements($customFieldProcessor)))`
instead of
`if ($customFieldProcessor instanceof FieldProcessor) `

When I add a fieldprocessor in a custom ext, instanceof will return for example
`VENDOR\MyExt\Solr\FieldProcessor\DaysBooleanToStrings`
and not
`ApacheSolrForTypo3\Solr\FieldProcessor\FieldProcessor`

Reply:
```
php -a
Interactive shell

php > interface Foo{}; class Bar implements Foo{}; $x = new Bar(); var_dump($x instanceof Foo);
bool(true)
```

Did you use `YourProcessor implements ApacheSolrForTypo3\Solr\FieldProcessor\FieldProcessor` ?

Reply:
This I use in my class def:
`class Waitlist implements \ApacheSolrForTypo3\Solr\FieldProcessor\FieldProcessor`